src/services/dashboard.js

```diff
@@ -76,13 +75,11 @@ export async function queryTocResult(params) {
 
   const indices = `${getAllMonthsWithinRange(
     endpoints,
-    endpoints.run_toc_index,
+    endpoints.indices.run_toc_index,
     selectedDateRange
   )}/_search`;
 
-  const endpoint = MOCK_UI
-    ? `${endpoints.pbench_server}/datasets/toc`
-    : `${endpoints.pbench_server}/elasticsearch`;
+  const endpoint = MOCK_UI ? endpoints.api.datasets_toc : endpoints.api.elasticsearch;
```

Comment:
Are these two endpoints really swappable?  And, why does the code need to know whether its dependency is being mocked?...can't we go with just one choice and mock that when we want to test?  (Ditto for lines 104 and 190, et al.)

Reply:
The mocks used to trigger on data within the Elasticsearch JSON payload, because the actual URIs are identical.

Gurbir switched to this model because it means each "API" has a distinct URI that can be reliably and straightforwardly mocked. Once we've replace the direct Elasticsearch calls with Pbench APIs, these splits will go a way. Right now, the production code needs to make the elasticsearch calls, but using the planned Pbench API URIs vastly simplifies the mocks.

Reply:
Only one payload, although that payload will change once we shift over to the backend API. Right now, the dashboard test environment isn't trying to pretend that `/api/v1/datasets/toc` is returning the fully post-processed data we anticipate for the Pbench API ... it's just the raw Elasticsearch output the dashboard effect currently expects. We're just faking the URI to make it easier to mock. Yeah, it's weird, but the old mocking code that was based on searches of the JSON query payload was even more weird. 😬 

And, again, it's only until we actually implement those APIs ...